### DISCUSSION TITLE
Feature Request: Schedule Deployment/Redeploy

### DISCUSSION THREAD

**Discussion by j471n** [vercel/community#3015] [Idea]
### Description
I would like to propose a new feature for Vercel that allows users to schedule deployments or redeployments of their projects. This feature would enable users to specify a specific date and time for their project to be deployed or redeployed automatically, without the need for manual intervention.

### Use Case
There are several scenarios where the ability to schedule deployments/redeployments would be beneficial:

1. **Scheduled releases:** It would be helpful to schedule deployments to occur at specific dates and times, especially for planned feature releases or updates.
2. **Off-peak hours:** For applications with a large user base, scheduling deployments during off-peak hours can minimize disruption and ensure a smoother user experience.
3. **Timezone considerations:** Users in different timezones may prefer to schedule deployments at specific times to align with their working hours or to ensure support availability during critical periods.
4. **Testing and staging environments:** Scheduling redeployments for testing or staging environments can streamline development workflows and automate the deployment process.

### Proposed Solution
I suggest adding a new option in the Vercel dashboard to enable scheduling deployments or redeployments. This could be implemented through a user-friendly interface where users can set the desired date and time for the deployment, along with any additional configuration options such as timezone selection.

### Benefits
- Increased flexibility and convenience for users by allowing them to schedule deployments in advance.
- Streamlined deployment workflows, particularly for recurring deployments or deployments across multiple environments.
- Reduced manual intervention and potential for human error by automating the deployment process.
- Improved user experience for applications with large user bases by scheduling deployments during off-peak hours.

### Additional Information
Please let me know if there are any specific technical considerations or limitations that should be taken into account for implementing this feature. I believe this feature would greatly enhance the deployment capabilities of Vercel and provide added value to its users.

_Note: I am not talking about a cron job or something, I am talking about this feature in the vercel dashboard._ 

Thank you for considering my request!

**Comment by leerob** [comment]
Thanks for the suggestion! In the meantime, as you mentioned, a [Vercel Cron Job](https://vercel.com/docs/cron-jobs) + using the [Vercel REST API](https://vercel.com/docs/rest-api) would allow you to kick off a deployment on a schedule 👍
